cmd/conformance/mysql/README.md

```diff
@@ -47,3 +47,29 @@ go run ./cmd/conformance/mysql --mysql_uri="root:root@tcp(localhost:3306)/test_t
 ### Stopping
 
 <kbd>Ctrl</kbd> <kbd>C</kbd>
+
+## Using the log
+
+In this example, we're running 32 writers against the log to add 128 new leaves within 1 minute.
+
+Note that the writes are sent to the HTTP server we brought up in the previous step, but reads are sent directly to the file system.
+
+```shell
+go run ./hammer \
+  --log_public_key=Test-Betty+df84580a+AQQASqPUZoIHcJAF5mBOryctwFdTV1E0GRY4kEAtTzwB \
+  --log_url=http://localhost:2024/ \
+  --max_read_ops=0 \
+  --num_writers=32 \
+  --max_write_ops=64 \
+  --max_runtime=1m \
+  --leaf_write_goal=128 \
```

Comment:
Try a larger number like `512`, so it will pass the `256` tile size.

Reply:
Or even bigger numbers like 1024? :-)